packages/eslint-plugin-svelte/src/rules/no-useless-children-snippet.ts

```diff
@@ -0,0 +1,29 @@
+import { createRule } from '../utils/index.js';
+
+export default createRule('no-useless-children-snippet', {
+	meta: {
+		docs: {
+			description: "disallow explicit children snippet where it's not needed",
+			category: 'Best Practices',
+			recommended: false
+		},
+		schema: [],
+		messages: {
+			uselessSnippet: 'Found an unnecessary children snippet.'
+		},
+		type: 'suggestion'
+	},
+	create(context) {
+		return {
+			SvelteSnippetBlock(node) {
+				if (
+					node.parent.type === 'SvelteElement' &&
```

Comment:
We need to check `<svelte:component>` also.

https://svelte.dev/playground/hello-world?version=5.16.0#H4sIAAAAAAAAE3VQwW7CMAz9FeMdAKmi96xDmnbhH-gO0BoRKbWjxLBNUf59SqGaitjNfn7v2X4J-TAQGtyRcwJfElwPK-qtUr_GCk_WUUSzT6g_vvAKgNWkevd-E6_ktGDHQ6RneCesxBrRYHMDjXi1whHChSlCvW255SZ2wXottdrBS1D4kMHDKcgAy01dGmFivTsvX1tu6j8RT-bdRAQ92_iWijCPtuklsvWeFLqzdX0gXq1zywAA4_-lTPWdk0f7B8stVqj0rWg0XChX_-TyeOs8nSfTWUZObins7GK8ofTzvZ_5Fwm0cgS5AQAA